### PR TITLE
Bug fix/adding handles

### PR DESCRIPTION
**Pull request by GabiDanciu**
#### What problem is this solving?

I needed some css handles. I had to stylize the option boxes with a background-color a border radius and I didn't have handles on second div children of  inputValueOptionBox div.  Also I added a handle when one box is selected.

#### How to test it?

https://product--floriaro.myvtex.com/ancora-de-pamant-pentru-troliere-ro/p
it's about those option boxes in the right main column of the product page. 

#### Screenshots or example usage:

image

### CODE REVIEW

File: docs/README.md
Comment (icazevedo) [inline docs/README.md:153]: This should be a modifier.

Use the `applyModifiers` function that is exported alongside the `useCssHandles` hook. Here's a [link to the docs](https://github.com/vtex-apps/css-handles/blob/7f8257d50a1d01977e4e06048790a0ea03003297/docs/README.md).

**Review by icazevedo — CHANGES_REQUESTED** [review]
Thanks for the PR! I've left a suggestion on how to improve the solution. Also, add a changelog entry.

File: docs/README.md
Comment (GabiDanciu) [reply docs/README.md]: done, please check againg

File: react/components/ProductAssemblyOptions/InputValue/OptionBox.tsx
Comment (icazevedo) [inline react/components/ProductAssemblyOptions/InputValue/OptionBox.tsx:35]:
```suggestion
          applyModifiers(handles.inputValueOptionBoxItem,  selected ? 'active' : ''),
```

**Review by icazevedo — CHANGES_REQUESTED** [review]
Please fix the lint errors 🙏

**Comment by GabiDanciu** [comment]
> Please fix the lint errors 🙏

Done, It is that ok?

File: CHANGELOG.md
Comment (icazevedo) [inline CHANGELOG.md:10]:
```suggestion
### Added
```

**Review by icazevedo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md
Comment (icazevedo) [reply CHANGELOG.md:10]: Can you just fix this tiny typo, please? I would do it myself, but I don't that have the permissions to commit on your branch

Comment (GabiDanciu) [reply CHANGELOG.md]: yes, sure. My mistake :D.

**Review by icazevedo — APPROVED** [review]

**Review by igorbrasileiro — APPROVED** [review]
lgtm

**Comment by vtexdaniel** [comment]
Same question here: @igorbrasileiro @icazevedo can this be merged please or do we need additional approvals?

Thanks!

File: CHANGELOG.md
Comment (victorhmp) [inline CHANGELOG.md:10]: This will solve the conflict pointed out by GitHub

```suggestion
## [Unreleased]
### Added
```

**Review by victorhmp — APPROVED** [review]

**Comment by igorbrasileiro** [comment]
Fix conflicts @GabiDanciu, please.

**Comment by razvanudream** [comment]
> Fix conflicts @GabiDanciu, please.

Fixed it for him.  Thanks and will keep an eye on this for when it gets merged.

**Comment by mihainutiu-vtex** [comment]
Created beta version until this PR is approved / merged / published

`vtex.product-customizer@2.11.3-beta.0`

**Comment by igorbrasileiro** [comment]
@mihainutiu-vtex hi, I'm no longer a vtex-apps maintainer.

**Pull request closed without merging**
